src/tree/PostgresAccountWizard/PostgresServerCredStep.ts

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { nonNullProp } from '../../utils/nonNull';
+import { IPostgresWizardContext } from './IPostgresWizardContext';
+
+export class PostgresServerCredStep extends AzureWizardPromptStep<IPostgresWizardContext> {
+
+    public async prompt(wizardContext: IPostgresWizardContext): Promise<void> {
+        wizardContext.adminUser = (await ext.ui.showInputBox({
+            placeHolder: "Username",
+            prompt: 'Enter administrator username for the server.',
+            validateInput: validateUser,
+        })).trim();
+        const user = nonNullProp(wizardContext, 'adminUser');
+        wizardContext.adminPassword = (await ext.ui.showInputBox({
```

Comment:
Each call to `ext.ui.showInputBox` has to be a separate prompt step otherwise the back button won't work properly